aws-account-operator-services/aws-account-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 13c36e34cec72a9e416b7a48f75cc7586f23c897
+  name: aws-account-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```

Comment:
this path doesn't seem to exist

Reply:
I have a branch in the aws-account-operator repo which contains the scripts just cleaning them up.

Reply:
That PR is ready for review/merge https://github.com/openshift/aws-account-operator/pull/38